xdsl/dialects/memref.py

```diff
@@ -484,20 +485,33 @@ def verify_(self) -> None:
                 "Global initial value is expected to be a "
                 "dense type or an unit attribute"
             )
+        if self.alignment is not None:
+            assert isa(self.alignment, IntegerAttr)
```

Comment:
Do we need the isa here? can we use isinstance?

Reply:
seems to work!